ironfish-cli/src/commands/debug.ts

```diff
@@ -33,6 +34,14 @@ export default class Debug extends IronfishCommand {
 
     const telemetryEnabled = this.sdk.config.get('enableTelemetry').toString()
 
+    let cmdInPath: boolean
+    try {
+      execSync('ironfish --help &>/dev/null')
```

Comment:
Is there a way we could get this to work on Windows as well? Or at least have a carve-out for Windows that just says "UNKNOWN". Maybe there's a way to run the command and check the error code, I'm not totally certain

Reply:
🤦 I changed to this implementation from my original one because it wasn't Windows compatible, then I went and added `/dev/null` anyway. Changed it to just have node ignore stdio for the child process which seems to work just the same